hydromt_wflow/components/__init__.py

```diff
@@ -1,6 +1,4 @@
-"""submodule for model components."""
+"""Wflow model components submodule."""
 
-from hydromt_wflow.components.config import WflowConfigComponent
-from hydromt_wflow.components.grid import WflowGridComponent
-
-__all__ = ["WflowConfigComponent", "WflowGridComponent"]
+from .config import *
+from .staticmaps import *
```

Comment:
I'm not a fan of star imports, especially if we didn't set the `__all__` because this is very bad for type checkers. Can we use more direct imports here?